cherrypy/test/test_bus.py

```diff
@@ -261,10 +261,19 @@ def g():  # pylint: disable=invalid-name
 
     # The last message will mention an indeterminable thread name; ignore
     # it
-    assert (log_tracker.log_entries[:-1] ==
-            ['Bus STOPPING', 'Bus STOPPED',
-             'Bus EXITING', 'Bus EXITED',
-             'Waiting for child threads to terminate...'])
+    expected = [
+        'Bus STOPPING',
+        'Bus STOPPED',
+        'Bus EXITING',
+        'Bus EXITED',
+        'Waiting for child threads to terminate...',
+    ]
+    # If the last message mentions an indeterminable thread name then ignore it
+    len_expected = len(expected)
+    assert log_tracker.log_entries[:len_expected] == expected
+    assert len(log_tracker.log_entries) - len_expected in (0, 1), (
```

Comment:
Why did you obfuscate the check? Using `len(log_tracker.log_entries[len_expected:]) < 2` is way cleaner

Reply:
This box is checked on this PR.
> Allow edits by maintainers

That means that maintainers have the ability to modify the code in this PR to match cherrypy’s needs.  If there are cleaner ways to express these tests then please feel free to make appropriate mods.  Our goal is to work together have a strong test suite that passes more than it fails.  This is required to give users and contributors confidence in the health of cherrypy.

Reply:
Sure, you are absolutely right. That's why when you do something that I didn't expect I first try to ask about the motivation. Instead of assuming that you've made a mistake, I assume that you may have some serious reason behind applying the change in a different way. And in such unobvious cases, I prefer to ask if that's intentional rather than forcefully override the submission.

Reply:
It may not seem like it but I'm very grateful for your effort to fix the CIs. It's something that I'm struggling to find time for lately.